rust/template/tcp_channel/message.rs

```diff
@@ -10,3 +14,15 @@ pub enum Message<T> {
     Commit,
     Complete,
 }
+
+impl<T> Display for Message<T> {
```

Comment:
I'm not sure this is a good `Display` implementation. In the future we may want a "proper" `Display` that shows all updates rather than just print the name of the command.

Reply:
I understand what you are saying, but please consider that this is solely an internal type (with no intention/reason of ever being exposed outside the crate) and we have full control over the few locations it is used in within this crate. I deliberately picked `Display` because `Debug` would be more appropriate for producing a dump that includes updates, should the need arise. In my opinion there is no wrong or right implementation of `Display` here. Yes, we could introduce a "to_type_str" method or something like that to avoid using `Display`, but at this point it feels as if that just adds boilerplate code.
Just my opinion, do let me know if you disagree and I'll change it, though.

Reply:
Ok, I did not realize this was internal to tcpchannel